### DISCUSSION TITLE
Synchronize X-axes of panels

### DISCUSSION THREAD

**Discussion by sciurius** [grafana/grafana#46616] [Dashboards, Panels & Visualizations]
Given a number of timeseries panels stacked vertically. Each of the panels has the same time region.
I want the graph part of the panels to be aligned such that the same x values are exactly on the same vertical line.

This is usually not possible, since the graphs can have different (widths) axis labels and legends (when legends are on the right). So the desired feature would mean that a) the panels have the same x-range and b) are aligned so that the ticks are straight under each other. 

The mock-up shows what I’m after. 

![scrot20220316082100](https://user-images.githubusercontent.com/1118804/158537280-26ac8211-ff8e-42ed-99f2-22a09b2adfa4.png)

**Comment by sciurius** [comment]
To prevent any optimism... https://github.com/grafana/grafana/issues/4469

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
